src/binder_devmon_if.c

```diff
@@ -240,15 +263,16 @@ static
 BinderDevmonIo*
 binder_devmon_if_start_io(
     BinderDevmon* devmon,
-    RadioClient* client,
+    RadioClient* ds_client,
```

Comment:
Seems to be unused?

Reply:
The way device monitoring is done it that there the binder_devmon_combine.c which handles starting both and therefore we need to provide both interfaces to both if and ds versions because combining code does not know which client to use for which. I know this is not very nice way but probably would have needed more refactoring to make it cleaner.

Reply:
So this comes as a requirement from being a callback? OK.